recipe/meta.yaml

```
@@ -24,9 +26,15 @@ requirements:
     - llvm  # [llvmdev not in ("5.*", "6.*", "7.*")]
     - llvmdev
     - clangdev
+  run_constrained:
+    - cling ==9999999999  # [clangdev != "5.*"]
+    - cling 0.8           # [clangdev == "5.*"]
 
 test:
+  requires:                # [clangdev == "5.*"]
+    - jupyter_kernel_test  # [clangdev == "5.*"]
   commands:
+    - python $RECIPE_DIR/jupyter_Clad_kernel_test.py
```

Apologies, I always forgot the right way of doing this: https://conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-files
```suggestion
  files:
    - jupyter_Clad_kernel_test.py
  commands:
    - python jupyter_Clad_kernel_test.py  # [clangdev == "5.*"]
```